apps/wallet-mobile/src/wallet-manager/types.ts

```diff
@@ -20,5 +20,15 @@ export type AddressMode = 'single' | 'multiple'
 export type WalletManagerEvent =
   | {type: 'easy-confirmation'; enabled: boolean}
   | {type: 'hw-device-info'; hwDeviceInfo: HWDeviceInfo}
+  | {type: 'selected-wallet-id'; id: YoroiWallet['id']}
 
 export type WalletManagerSubscription = (event: WalletManagerEvent) => void
+
+export type WalletInfo = {
```

Comment:
this can be storage later and when restoring -> syncing it can be used to notify or badge the wallet with new txs, new rewards, etc...